TemporaryEmail/source/info.plist

```diff
@@ -64,7 +68,20 @@ fi
 email="${email_name}@teleosaurs.xyz"
 url="https://teleosaurs.xyz/inbox/${email_name}"
 
-front_browser="$(osascript -e 'tell application "System Events" to return name of first process whose frontmost is true')"
+get_front_browser
```

Comment:
Making this a function instead of a one-liner has no advantage if it’s only called once. It’s making one line into four (including the original line verbatim), spread over two sections. It also sets a global variable inside a function, which isn’t good practice.